src/openms/source/METADATA/AbsoluteQuantitationStandards.cpp

```diff
@@ -84,7 +84,10 @@ namespace OpenMS
         }
         if (run.IS_component_name != "")
         {
-          findComponentFeature_(fmap, run.IS_component_name, fc.IS_feature);
+          if (!findComponentFeature_(fmap, run.IS_component_name, fc.IS_feature))
+          { // down stream methods will expect to find an IS
+            continue;
```

Comment:
should a warning or some debug message be written?

Reply:
Good idea.